#include "team_client_player_replied_handler.h"
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
void ClientPlayerTeamServiceRepliedHandler::TeamInfoNotify(entt::entity player,
	const ::TeamInfoS2CRequest* request,
	::TeamInfoS2CResponse* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
}

